src/main/resources/openid-connect.yaml

```diff
@@ -42,6 +42,42 @@ paths:
         401:
           description: Username password combination not found
 
+  /authenticate?service=otpe:
```

Comment:
Might have been easier to use /authenticate/otp/

Reply:
we can discuss that with Krem and maybe get this updated
I agree it's probably easier to guess